org.jacoco.core.test.validation/pom.xml

```diff
@@ -108,13 +104,9 @@
           <value>7</value>
         </property>
       </activation>
-      <properties>
-        <groovy.targetBytecode>1.7</groovy.targetBytecode>
-      </properties>
       <modules>
         <module>../org.jacoco.core.test.validation.kotlin</module>
         <module>../org.jacoco.core.test.validation.java7</module>
-        <module>../org.jacoco.core.test.validation.groovy</module>
```

Comment:
I was wondering about the statement

> As Java 7 is not supported any more therefore Groovy tests are now executed for >= Java 8

"not supported" by whom? Apparently it is about Groovy 3 - https://groovy-lang.org/releasenotes/groovy-3.0.html

> Groovy 3.0 requires JDK9+ to build and JDK8 is the minimum version of the JRE

then in `org.jacoco.doc/docroot/doc/build.html` in row `org.jacoco.core.test.validation.groovy` we should replace `7` by `8` and empty cell about `JDK 7`.